dvc/objects/db/base.py

```diff
@@ -42,7 +42,7 @@ def get(self, hash_info):
             hash_info,
         )
 
-    def add(self, path_info, fs, hash_info, **kwargs):
+    def add(self, path_info, fs, hash_info, move=True, **kwargs):
```

Comment:
For the record: `move` option is a bit clunky (same as `isinstance(fs, type(self.fs))`) but we kinda have to keep it for now, until we migrate to fsspec and will be able to distinguish rename/move copy/servercopy.